### DISCUSSION TITLE
For writing Retrospective

### DISCUSSION THREAD

**Discussion by hitomipupil** [HYF-Class22/agile-development-group3-hyf-redesign#30] [Ideas]
# Retrospective

## Stop Doing
-  Infrequent communication

## Continue Doing
- note down what we discussed during the meeting
- Implement a clear structure in the HTML file before starting work on collaboration
- Communicate any problems and challenges to other team members
- Give each other feedback to improve work

## Start Doing
- use Discussions on gitHub
- Learn new thing together and share what you have learned
- Prioritize realistic timelines for tasks 
- Provide feedback

## Lessons Learned
- strong and clear plans for projects makes tasks easier.
- a clear plan also served as a good communication tool
- adding class/id to HTML tag avoids conflicts
---

### Or remove extra steps?

**Comment by jgchoti** [comment]
## Strategy vs. Board

### What parts of your plan went as expected?
- A well-structured HTML file is more readable, making it easier for team members to navigate 
### What parts of your plan did not work out?
- Elements in HTML and CSS need more consistent naming and better organized 
- Limited coding skills have contributed to time-consuming tasks

### Did you need to add things that weren't in your strategy?
- Reorganizing code for improved consistency 

### Or remove extra steps?
